package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "rexpack",
+  "version": "1.0.0",
+  "description": "Expack is the bare-bones Express and Webpack boilerplate with ES6+ babel transpilation, ESLint linting, Hot Module Reloading, and Jest test framework enabled.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "build": "rm -rf dist && webpack --mode development --config ./config/webpack/webpack.server.config.js && webpack --mode development --config ./config/webpack/webpack.dev.config.js",
+    "prod": "rm -rf dist && webpack --mode production --config ./config/webpack/webpack.server.config.js && webpack --mode production --config ./config/webpack/webpack.prod.config.js",
+    "start": "node ./build/server.js",
+    "lint": "eslint . --fix"
+  },
+  "repository": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "express": "^4.16.4",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-test-renderer": "^17.0.1"
+  },
+  "engines": {
+    "node": ">=4.3.2"
+  },
+  "jest": {
```

Comment:
lets move it to separate file jest.config.js

Reply:
done